casper/src/test/scala/coop/rchain/casper/genesis/PosMultiSigTransferSpec.scala

```diff
@@ -48,6 +54,7 @@ class PosMultiSigTransferSpec extends FlatSpec with Matchers with Inspectors {
           numberOfActiveValidators = 100,
           validators = bonds.map(Validator.tupled).toSeq,
           posMultiSigPublicKeys = Seq(pub1, pub2, pub3).map(p => p.bytes.toHexString).toList,
+          posVaultPubKey = genesisPubKey.bytes.toHexString,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.